src/main/java/ai/rev/speechtotext/AsyncApiInterface.java

```diff
@@ -16,7 +25,7 @@
  * The ApiInterface is a type-safe Retrofit interface that presents all the endpoints that are made
  * to communicate with the Rev.AI API.
  */
-public interface ApiInterface {
+public interface AsyncApiInterface {
```

Comment:
Since this is public access changing the name of this interface would be a breaking change, right?